sos/plugins/ceph.py

```diff
@@ -132,4 +165,29 @@ def setup(self):
             "/etc/ceph/*bindpass*"
         ])
 
+        # If containerized, run commands in containers
+        if self.ceph_containers:
+            # Avoid retrieving multiple times the same data
+            got_ceph_cmds = False
+            for container in self.ceph_containers:
+                if re.match("ceph-(mon|rgw|osd)", container[1]) and \
+                        not got_ceph_cmds:
+                    self.add_cmd_output([
+                        self.fmt_container_cmd(container[1], "ceph %s" % s)
+                        for s in ceph_cmds
+                    ])
```

Comment:
`Plugin.fmt_container_cmd()` does not exist in `legacy-3.9`. It was added in as part of the 4.0 update.

Reply:
Ah, good catch, the original PR is a trap aside a trap :)